src/locale/locales/nl/messages.po

```diff
@@ -0,0 +1,7903 @@
+
+#: src/screens/Onboarding/StepProfile/AvatarCreatorItems.tsx:107
+msgid "Choose this color as your avatar"
+msgstr "Kies deze kleur als uw avatar"
+
+#: src/screens/Signup/StepInfo/index.tsx:201
+msgid "Choose your password"
+msgstr "Kies uw wachtwoord"
+
+#: src/view/screens/Settings/index.tsx:877
+msgid "Clear all storage data"
+msgstr "Wis alle opslaggegevens"
+
+#: src/view/screens/Settings/index.tsx:880
+msgid "Clear all storage data (restart after this)"
+msgstr "Wis alle opslaggegevens (hierna opnieuw opstarten)"
+
+#: src/components/forms/SearchInput.tsx:70
+msgid "Clear search query"
+msgstr "Wis zoekopdracht"
+
+#: src/view/screens/Settings/index.tsx:878
+msgid "Clears all storage data"
+msgstr "Wist alle opslaggegevens"
+
+#: src/view/com/modals/DeleteAccount.tsx:216
+msgid "Click here for more information."
+msgstr "Klik hier voor meer informatie."
+
+#: src/view/com/modals/DeleteAccount.tsx:208
+msgid "Click here for more information on deactivating your account"
+msgstr "Klik hier voor meer informatie over het deactiveren van uw account"
+
+#: src/components/TagMenu/index.web.tsx:152
+msgid "Click here to open tag menu for {tag}"
+msgstr "Klik hier om het tagmenu voor {tag} te openen"
+
+#: src/components/dialogs/PostInteractionSettingsDialog.tsx:304
+msgid "Click to disable quote posts of this post."
+msgstr "Klik om citaatberichten voor dit bericht uit te schakelen."
+
+#: src/components/dialogs/PostInteractionSettingsDialog.tsx:305
+msgid "Click to enable quote posts of this post."
+msgstr "Klik om citaatberichten voor dit bericht in te schakelen."
+
+#: src/components/dms/MessageItem.tsx:240
+msgid "Click to retry failed message"
+msgstr "Klik om mislukt privébericht opnieuw te proberen"
+
+#: src/screens/Onboarding/index.tsx:32
+msgid "Climate"
+msgstr "Klimaat"
+
+#: src/components/dms/ChatEmptyPill.tsx:39
+msgid "Clip 🐴 clop 🐴"
+msgstr "Clip 🐴 clop 🐴"
+
+#: src/components/dialogs/GifSelect.tsx:282
+#: src/components/dialogs/VerifyEmailDialog.tsx:246
+#: src/components/dms/dialogs/SearchablePeopleList.tsx:263
+#: src/components/NewskieDialog.tsx:146
+#: src/components/NewskieDialog.tsx:153
+#: src/components/StarterPack/Wizard/WizardEditListDialog.tsx:117
+#: src/components/StarterPack/Wizard/WizardEditListDialog.tsx:123
+#: src/view/com/modals/ChangePassword.tsx:268
+#: src/view/com/modals/ChangePassword.tsx:271
+#: src/view/com/util/post-embeds/GifEmbed.tsx:197
+msgid "Close"
+msgstr "Sluiten"
+
+#: src/components/dialogs/GifSelect.tsx:169
+msgid "Close GIF dialog"
+msgstr "Sluit GIF-dialoogvenster"
+
+#: src/components/Dialog/index.web.tsx:110
+#: src/components/Dialog/index.web.tsx:256
+msgid "Close active dialog"
+msgstr "Sluit actief dialoogvenster"
+
+#: src/screens/Login/PasswordUpdatedForm.tsx:32
+msgid "Close alert"
+msgstr "Sluit waarschuwing"
+
+#: src/view/com/util/BottomSheetCustomBackdrop.tsx:36
+msgid "Close bottom drawer"
+msgstr "Sluit onderste lade"
```

Comment:
Right. I don't think we would call the bottom drawer the onderste lade. It's "the drawer that shows from the bottom" not the "lowest drawer".

I don't have a good suggestion yet, but this needs to be revised.

Reply:
Do you happen to know more context what it relates to?

Reply:
I think it's an aria-label for a UI element that is shown from the bottom:

![Android example of a Bottom sheet](https://github.com/user-attachments/assets/fcdc09de-6d7b-4311-bdfe-3d4da697f03f)
![Apple examples of a Bottom sheet](https://github.com/user-attachments/assets/d3d04082-4826-4ada-9e72-c54cd0f91305)

...but I have not found how to call it in the app. 

That piece of paper that appears from the bottom (aka bottom sheet) is not a drawer, which appears from the side (generally the menu and sidebar that float over content are called drawers).

Reply:
OK, thanks! I have changed it to the original word: "Sluit onderste drawer" with argumentation that it is a concept, for which no accepted translation exists in Dutch. Might not be true ultimately, but a screenreader using the source might be able to understand the translation too. In case it is a bug in the UI English, it has not yet received enough attention to cause a correction.